controlplane/pkg/tests/nsmd_plugin_test.go

```diff
@@ -0,0 +1,61 @@
+package tests
+
+import (
+	"context"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	pluginsapi "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/plugins"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/plugins"
+)
+
+func TestPluginReRegistration(t *testing.T) {
+	g := NewWithT(t)
+	var err error
+
+	const pluginName = "dummy-plugin"
+
+	registryDir := path.Join(os.TempDir(), "networkservicemesh")
+	if err = os.MkdirAll(registryDir, os.ModeDir|os.ModePerm); err != nil {
+		logrus.Errorf("Failed to create socket folder for Plugin Registry: %v", err)
+		return
+	}
+	defer func() { _ = os.RemoveAll(registryDir) }()
+
+	registryPath := path.Join(registryDir, path.Base(pluginsapi.PluginRegistrySocket))
+
+	pluginRegistry := plugins.NewPluginRegistry(registryPath)
+	if err = pluginRegistry.Start(context.Background()); err != nil {
+		logrus.Errorf("Failed to start Plugin Registry: %v", err)
```

Comment:
`logrus.Errorf` doesn't fail the test. We should explicitly call `t.Fatalf("...reason...")` or something like that

Reply:
- [ ] logrus.Errorf => logrus.Fatalf